packages/amplication-server/src/core/resource/resource.service.ts

```diff
@@ -659,9 +569,9 @@ export class ResourceService {
     });
   }
 
-  async workspace(resourceId: string): Promise<Workspace> {
-    return await this.prisma.resource
-      .findUnique({ where: { id: resourceId } })
+  async project(projectId: string): Promise<Project> {
+    return this.prisma.project
+      .findUnique({ where: { id: projectId } })
       .workspace();
   }
```

Comment:
This is not the expected behavior...

Previously it was returning a workspace by a resource ID - I would expect it to return a project by resource ID... instead if return a workspace by project ID

Reply:
@yuval-hazaz I fixed it.
This method is used in `resource.resolver.ts`